pom.xml

```diff
@@ -497,27 +498,13 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
-            <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-bom</artifactId>
-                <version>2.24.3</version>
-                <scope>import</scope>
-                <type>pom</type>
-            </dependency>
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
                 <version>5.12.1</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-            <dependency>
-                <groupId>org.slf4j</groupId>
-                <artifactId>slf4j-bom</artifactId>
```

Comment:
Using bom allows you to set the version only once.

Reply:
i know, but it also introduces some indirection (i.e. _"pff... no version here, where is the version coming from, gaddamit"_). since, all logging stuff will come shortly from `spring-boot-starter-logging` i opted for simplicity and removing code.